src/js/controllers/modals/search.js

```diff
@@ -29,6 +29,7 @@ angular.module('copayApp.controllers').controller('searchController', function($
         var message = tx.message ? tx.message : '';
         var comment = tx.note ? tx.note.body : '';
         var addressTo = tx.addressTo ? tx.addressTo : '';
+        if (tx.amountUnitStr === 'BCH' && addressTo) addressTo = 'bitcoincash:'+addressTo;
```

Comment:
I believe addressTo is currently in the legacy format, so adding the prefix to it does not automatically change it to a CashAddr format. Perhaps you could do it this way: if it's BCH, you could append one more string to this searchable string, and the new string would be a proper CashAddr format. That way, the searchable string will have both the legacy format and the CashAddr format so users can search it in both formats. You can use the bitcoincashjs library (already part of the wallet project) to do the conversion.